src/Employer/Employer.Web/Views/Part1/Duration/Duration.cshtml

```diff
@@ -32,7 +32,7 @@
                     <span esfa-validation-message-for="Duration" class="govuk-error-message"></span>
                     <input asp-for="Duration" class="govuk-input govuk-input--width-2" type="number" min="1" >
                     <select asp-for="DurationUnit" class="govuk-select duration-unit-select">
-                        <option value="@DurationUnit.Month">Month(s)</option>
+                        <option value="@DurationUnit.Month">Months</option>
                         <option value="@DurationUnit.Year">Year(s)</option>
```

Comment:
why not make the same change for Years too?

Reply:
Apprenticeships are minimum 12 Months long so the optional (s) doesn't make sense but they could choose to write that as 1 Year.

Reply:
👍